include/constants/skills.person.enum.txt

```diff
@@ -21,3 +21,14 @@ SID_Drought
 SID_Drizzle
 SID_SnowWarning
 SID_SandStream
+
+// Initiative skills
+SID_InitCalm
+SID_InitClever
+SID_InitDeft
+SID_InitLucky
+SID_InitNimble
+SID_InitQuick
+SID_InitSpectrum
+SID_InitStrong
+SID_InitStrudy
```

Comment:
You should also note that as a requirement of code style, there should be a blank line at the end of the file. just the same as https://www.kernel.org/doc/html/v4.10/process/coding-style.html